src/main/java/org/broadinstitute/hellbender/tools/GetSampleName.java

```diff
@@ -17,13 +17,13 @@
 import java.util.stream.Collectors;
 
 @DocumentedFeature
+@ExperimentalFeature
 @CommandLineProgramProperties(
-        oneLineSummary = "(EXPERIMENTAL) Emit a single sample name from the bam header into an output file.",
```

Comment:
Did you go through the codebase and replace all instances of hardcoded `(EXPERIMENTAL)` text with the new annotation? I only see this one, and I know there were more. If you haven't, please do so in this PR.

Reply:
No, I only did the ones I was already touching. I'll search for any others and repair.